Mage.Sets/src/mage/cards/k/KarlovWatchdog.java

```diff
@@ -0,0 +1,81 @@
+package mage.cards.k;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.AttacksWithCreaturesTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.ContinuousRuleModifyingEffectImpl;
+import mage.abilities.effects.common.continuous.BoostControlledEffect;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.SubType;
+import mage.abilities.keyword.VigilanceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Zone;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+
+/**
+ *
+ * @author DominionSpy
+ */
+public final class KarlovWatchdog extends CardImpl {
+
+    public KarlovWatchdog(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{3}{W}");
+
+        this.subtype.add(SubType.DOG);
+        this.power = new MageInt(3);
+        this.toughness = new MageInt(2);
+
+        // Vigilance
+        this.addAbility(VigilanceAbility.getInstance());
+
+        // Permanents your opponents control can't be turned face up during your turn.
+        this.addAbility(new SimpleStaticAbility(Zone.BATTLEFIELD, new KarlovWatchdogEffect()));
+
+        // Whenever you attack with three or more creatures, creatures you control get +1/+1 until end of turn.
+        this.addAbility(new AttacksWithCreaturesTriggeredAbility(
+                new BoostControlledEffect(1, 1, Duration.EndOfTurn), 3));
+    }
+
+    private KarlovWatchdog(final KarlovWatchdog card) {
+        super(card);
+    }
+
+    @Override
+    public KarlovWatchdog copy() {
+        return new KarlovWatchdog(this);
+    }
+}
+
+class KarlovWatchdogEffect extends ContinuousRuleModifyingEffectImpl {
+
+    KarlovWatchdogEffect() {
+        super(Duration.WhileOnBattlefield, Outcome.Benefit);
+        staticText = "Permanents your opponents control can't be turned face up during your turn";
+    }
+
+    private KarlovWatchdogEffect(final KarlovWatchdogEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public KarlovWatchdogEffect copy() {
+        return new KarlovWatchdogEffect(this);
+    }
+
+    @Override
+    public boolean checksEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.TURN_FACE_UP;
+    }
+
+    @Override
+    public boolean applies(GameEvent event, Ability source, Game game) {
+        return game.isActivePlayer(source.getControllerId()) &&
+                game.getOpponents(source.getControllerId()).contains(event.getPlayerId());
```

Comment:
I don't think this is right. You want to check the controller of the object, and that's not necessarily the same as `event.getPlayerId()` if you check how it is used (unfortunately no description in GameEvent.EventType so have to look at where it's fired in `CardImpl::turnFaceUp`)

As coded here, this corresponds to "your opponents can't turn permanents face up during your turn" so if someone chose to play Break Open then the interaction would be incorrect